patches/tock/09-add-vendor-hid-usb-interface.patch

```diff
@@ -0,0 +1,331 @@
+diff --git a/boards/nordic/nrf52840dk_opensk/Cargo.toml b/boards/nordic/nrf52840dk_opensk/Cargo.toml
```

Comment:
The `boards` don't need to be part of the patch. We copy them into Tock during setup, and you can apply and commit your changes to the top level `boards` directory.

Reply:
Thanks.
Done.